roles/edpm_neutron_dhcp/templates/wrappers/dnsmasq.j2

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+{% if edpm_neutron_dhcp_sidecar_debug -%}
+set -x
+{%- endif -%}
+
+ARGS="$@"
+
+IMAGE_NAME={{ edpm_neutron_dhcp_sidecar_dnsmasq_image_name }}
+NEUTRON_CONFIG_DIR={{ edpm_neutron_dhcp_agent_config_dir }}
+NEUTRON_LIB_DIR={{ edpm_neutron_dhcp_agent_lib_dir }}
+
+{% raw -%}
+# Extract the network namespace UUID from the command line args provided by
+# neutron. Typically of the form (with dnsmasq as an example):
+#
+# dnsmasq --no-hosts --no-resolv --except-interface=lo \
+#   --pid-file=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/pid  \
+#   --dhcp-hostsfile=/var/lib/neutron/dhcp/317716b8-919a-4a6f-8db1-78128ec3b100/host ...
+NETNS=$(ip netns identify)
+NAME=neutron-dnsmasq-${NETNS}
+
+CLI="nsenter --net=/run/netns/${NETNS} --preserve-credentials -m -t 1 podman"
```

Comment:
I think --net= may not be necessary here since we are already inside the netns (as can be seen from line 20 and inspection of the logic here).